src/server/frontend/wayland/wayland_connector.cpp

```diff
@@ -87,6 +89,18 @@ namespace mir
 {
 namespace frontend
 {
+struct NullWlMirWindow : WlMirWindow
+{
+    void new_buffer_size(mir::geometry::Size const& ) {}
+    void commit() {}
+    void visiblity(bool ) {}
+    void destroy() {}
+
+    static NullWlMirWindow instance;
+} NullWlMirWindow::instance;
```

Comment:
Huh. This looks odd (but is correct!)

I think the idiomatic way to do this in the Mir codebase would be to have a NullWlMirWindow instance in an anonymous namespace?

Reply:
It looks odd because it is overcomplicated: There's no need for a static member variable. Vis:

```
namespace
{
struct NullWlMirWindow : mf::WlMirWindow
{
    void new_buffer_size(mir::geometry::Size const& ) {}
    void commit() {}
    void visiblity(bool ) {}
    void destroy() {}

} null_wl_mirwindow_instance;
}

mf::WlMirWindow* const mf::nullWlMirWindowPtr = &null_wl_mirwindow_instance;
...
```